feature_engine/features/within_person_discursive_range.py

```diff
@@ -12,7 +12,7 @@
 def get_nan_vector(vector_directory):
     current_script_directory = Path(__file__).resolve().parent
     # TODO --- fix this file path once dataset cleaning is added!
-    nan_vector_file_path = current_script_directory.parent / vector_directory / "nan_vector.txt"
+    nan_vector_file_path = current_script_directory.parent / "../feature_engine/tpm-data/vector_data/" / "nan_vector.txt"
```

Comment:
Wait, why are we hardcoding this instead of using the parameter `vector_directory`, which we are passing in?

Reply:
Oh, I see, it's because it lives in a specific spot, and we shouldn't expect users to put it into their vector directory! Makes sense.

Reply:
Updated, but note that we will eventually have to change this: https://github.com/Watts-Lab/team-process-map/issues/230